src/common/common-md-flag/common-md-flag.vcxproj

```diff
@@ -89,6 +89,7 @@
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
       <MultiProcessorCompilation Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">true</MultiProcessorCompilation>
       <MultiProcessorCompilation Condition="'$(Configuration)|$(Platform)'=='Release|x64'">true</MultiProcessorCompilation>
+      <DebugInformationFormat Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">ProgramDatabase</DebugInformationFormat>

```

Comment:
Is this a standalone fix or is it related to other fixes? I tried on current stable branch and I don't see any warning related to this change.

Reply:
It's a standalone fix. I've got warning `LNK4075: ignoring '/EDITANDCONTINUE' due to '/INCREMENTAL:NO' specification`

Reply:
We need to verify if it's correct that this project doesn't support incremental build.

Reply:
Let's remove this fix for now and merge the remaining fix for the introp lib